flutter-intellij.iml

```diff
@@ -28,5 +28,23 @@
         <SOURCES />
       </library>
     </orderEntry>
+    <orderEntry type="module-library" scope="PROVIDED">
+      <library>
+        <CLASSES>
+          <root url="jar://$MODULE_DIR$/lib/dart-plugin/163.10154.19/resources_en.jar!/" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+      </library>
+    </orderEntry>
+    <orderEntry type="module-library" scope="PROVIDED">
+      <library>
+        <CLASSES>
+          <root url="jar://$MODULE_DIR$/lib/dart-plugin/163.10154.19/snakeyaml-1.13.jar!/" />
```

Comment:
I wonder if we couldn't move these dependent jars our of the specific plugin directories so that we don't need to copy or re-up them when we add new plugin versions.

For example,

    lib/dart-plugin/
      deps/
        snakeyaml-1.13.jar
      163.10154.19/
        Dart.jar  

or something similar?